src/core/crypto/relay_crypto.c

```diff
@@ -100,12 +100,20 @@ relay_crypto_get_sendme_digest(relay_crypto_t *crypto)
   return crypto->sendme_digest;
 }
 
-/** Record the b_digest from <b>crypto</b> and put it in the sendme_digest. */
+/** Record the cell digest, indicated by is_foward_digest or not, as the
+ * SENDME cell digest. */
 void
-relay_crypto_record_sendme_digest(relay_crypto_t *crypto)
+relay_crypto_record_sendme_digest(relay_crypto_t *crypto,
+                                  bool is_foward_digest)
 {
+  struct crypto_digest_t *digest;
+
   tor_assert(crypto);
-  crypto_digest_get_digest(crypto->b_digest, (char *) crypto->sendme_digest,
+
+  (is_foward_digest) ? (digest = crypto->f_digest) :
```

Comment:
Please just use "if" for this kind of conditional.  Or use `digest = is_forward_digest ? __ : __;`

Reply:
Fixup commit: 5298b96783]